MainApp/Entities/Controllers/HeroController.cs

```diff
@@ -11,16 +11,32 @@ public class HeroController
         private MovementDeckController _movementDeckController;
         private GameFieldController _gameFieldController;
 
+        public bool WasLastMoveSuccessful { get; private set; }
```

Comment:
WasLastMove - how? Successfully or Unsuccessfully! It is an adverb, my friend.

Reply:
Thats not true, check TG for explanation